public/server/db.json

```
@@ -70,6 +70,45 @@
     "lol":{
         "definition": "Laugh Out Loud"
     },
+
+    "stfu":{
+        "definition": "Shut The Fuck Up"
+    },
+
+    "ngl":{
+        "definition": "Not Gonna Lie"
+    },
+
+    "hru":{
+        "definition": "How Are You"
+    },
+
+    "srly":{
+        "definition": "Seriously"
+    },
+
+    "bff":{
+        "definition": "Best Friend Forever"
+    },
+
+    "tl":{
+        "definition": "Time Line"
```

I believe `Time Line` is one word
```suggestion
        "definition": "Timeline"
```